Sources/Receipt/ReceiptDetailTableViewController.swift

```diff
@@ -51,11 +51,7 @@ public final class ReceiptDetailTableViewController: UITableViewController {
         tableView.allowsSelection = false
         tableView.rowHeight = UITableView.automaticDimension
         tableView.estimatedRowHeight = Theme.Cell.extraSmallHeight
-        tableView.separatorInset = UIEdgeInsets(
-            top: 0,
-            left: tableView.separatorInset.left,
-            bottom: 0,
-            right: .greatestFiniteMagnitude)
+        tableView.separatorStyle = .none
```

Comment:
Need to confirm with Willem and @jsaini-hw and see if this is fine, otherwise we need to add border for the sections.